<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="shhinduj-p51" ignored="0" name="Settings" tests="12" failures="3" timestamp="25 Apr 2019 07:03:47 GMT" time="130.359" errors="0">
  <testcase name="ta_41" time="38.628" classname="org.tibco.automation.common.testcases.Settings_Tests"/>
  <testcase name="ta_41" time="5.558" classname="org.tibco.automation.common.testcases.Settings_Tests"/>
  <testcase name="ta_41" time="6.743" classname="org.tibco.automation.common.testcases.Settings_Tests"/>
  <testcase name="ta_42" time="5.728" classname="org.tibco.automation.common.testcases.Settings_Tests"/>
  <testcase name="ta_42" time="6.858" classname="org.tibco.automation.common.testcases.Settings_Tests"/>
  <testcase name="ta_42" time="6.847" classname="org.tibco.automation.common.testcases.Settings_Tests"/>
  <testcase name="ta_43" time="6.857" classname="org.tibco.automation.common.testcases.Settings_Tests"/>
  <testcase name="ta_43" time="6.726" classname="org.tibco.automation.common.testcases.Settings_Tests"/>
  <testcase name="ta_43" time="6.757" classname="org.tibco.automation.common.testcases.Settings_Tests"/>
  <testcase name="ta_44" time="9.547" classname="org.tibco.automation.common.testcases.Settings_Tests">
    <failure type="java.lang.AssertionError" message="XSS Found">
      <![CDATA[java.lang.AssertionError: XSS Foundat org.tibco.automation.common.testcases.Settings_Tests.ta_44(Settings_Tests.java:110)... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- ta_44 -->
  <testcase name="ta_44" time="9.331" classname="org.tibco.automation.common.testcases.Settings_Tests">
    <failure type="java.lang.AssertionError" message="XSS Found">
      <![CDATA[java.lang.AssertionError: XSS Foundat org.tibco.automation.common.testcases.Settings_Tests.ta_44(Settings_Tests.java:110)... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- ta_44 -->
  <testcase name="ta_44" time="9.439" classname="org.tibco.automation.common.testcases.Settings_Tests">
    <failure type="java.lang.AssertionError" message="XSS Found">
      <![CDATA[java.lang.AssertionError: XSS Foundat org.tibco.automation.common.testcases.Settings_Tests.ta_44(Settings_Tests.java:110)... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- ta_44 -->
</testsuite> <!-- Settings -->
